CardView: add Getting Started (#7439)
DevExpress/devextreme-documentation: concepts/05 UI Components/CardView/05 Getting Started with CardView/00 Getting Started with CardView.md

#include tutorials-intro-installationnote

> CardView is a UI component that allows you to display data using cards. You ca use it for employee directories, contact lists, product catalogs, or task boards.

Reviewer: ```suggestion
CardView is a UI component that enables you to display data in a card-based format. You can use it for employee directories, contact lists, product catalogs, or task boards.
```

> This tutorial guides you how to add a CardView to a page and configure the component's core settings. 

Reviewer: ```suggestion
This tutorial explains how to add a CardView to a page and configure the component's core settings. 
```
